code/modules/ambient_occlusion/ao_verbs.dm

```diff
@@ -0,0 +1,22 @@
+/client/proc/global_ao_regenerate()
+	set name = "Regenerate AO (Global)"
+	set desc = "Regenerates AO caches across the map."
+	set category = "Debug"
+
+	if (!check_rights(R_DEBUG)) return
+
+	log_and_message_admins("has triggered a global ambient occlusion rebuild.")
+	usr << "Beginning global AO rebuild."
+
+	SSocclusion.disable()
+
+	for (var/turf/T in world)	// Yes, in world.
```

Comment:
In theory, because this loop is delayed, should you check for null turfs each iteration?

Although, can turfs ever be null. 🤔

Reply:
`for (var/turf/T)` can't select null values anyways; `null` is not a type of `/turf`.